package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "yarn lint:schemas && yarn lint:test-schemas && yarn lint:ajv",
+    "start": "cd ajv && DEBUG=ajv-schema-validator node app.js &",
```

Comment:
Any reason you used start, it feels like it should be some ajv ish?

Reply:
node apps usually follow convention as `start` for starting server hence the name. Happy to change if required

Reply:
I am guessing it does run node app.js :thinking:

Reply:
Yes it does `start` node server

Reply:
- Can we make this work without checking out to `ajv` dir?. Seems wrong for a command to change my working dir.
- I don't think we should run it in the background, I think it's fine for a server to be blocking the terminal. Not a big deal, but if we are keeping that, then we should add a stop command and document that.

Reply:
If it blocks the terminal then flask can't run. Flask has dependency on AJV service. I will add stop command and update readme